netkan/netkan/scheduler.py

```diff
@@ -22,25 +56,17 @@ def __init__(self, path, queue, base='NetKAN/'):
     def netkans(self):
         # This can easily be recursive with '**/*.netkan', however
         # implemeneting like for like initially.
-        return self.path.glob('*.netkan')
-
-    def generate_netkan_message(self, filename):
-        content = Path(filename).read_text()
-        return {
-            'Id': filename.stem,
-            'MessageBody': content,
-            'MessageGroupId': '1',
-            'MessageDeduplicationId': md5(content.encode()).hexdigest()
-        }
+        return (Netkan(f) for f in self.path.glob('*.netkan'))
```

Comment:
This is definitely a nit pick (we're only dealing with 1700 odd items here), but we're instantiating all these items into memory up front. Likely a little better to do it just before we need them.

Reply:
Are we? I looked that up and it should be a generator with the parentheses instead of brackets.

Reply:
![image](https://user-images.githubusercontent.com/1559108/65573967-a4b73980-df5b-11e9-8563-cec1df176c25.png)

Reply:
Well there you go! I missed that, I'll get another coffee ☕